spring-boot-project/spring-boot/src/test/java/org/springframework/boot/SpringApplicationTests.java

```diff
@@ -604,6 +605,17 @@ void addProfilesOrder() {
 		assertThat(environment.getActiveProfiles()).containsExactly("bar", "spam", "foo");
 	}
 
+	@Test
+	void includeProfilesOrder() {
+		SpringApplication application = new SpringApplication(ExampleConfig.class);
+		application.setWebApplicationType(WebApplicationType.NONE);
+		ConfigurableEnvironment environment = new StandardEnvironment();
+		application.setEnvironment(environment);
+		this.context = application.run("--spring.profiles.active=bar,spam", "--spring.profiles.include=foo");
+		// Since Boot 2.4 included profiles should always be last
+		assertThat(environment.getActiveProfiles()).containsExactly("bar", "spam", "foo");
+	}
```

Comment:
As my above reply, Could I add this test to another PR?

Reply:
I think we already cover the active and include case in [this test](https://github.com/spring-projects/spring-boot/blob/65c00f373a5520bac27a7b9a2fc99535189f5756/spring-boot-project/spring-boot/src/test/java/org/springframework/boot/context/config/ConfigFileApplicationListenerTests.java#L452).

Reply:
@mbhave 
In this test case, I want to test active and included profiles when using Boot 2.4 or later without using legacy processing.
It seems that the order of the active and included profiles has to be changed since boot 2.4 or later.
Could I add this test to another PR?